library/VTKExtensions/Rendering/vtkF3DRenderer.h

```diff
@@ -11,6 +11,8 @@
 #ifndef vtkF3DRenderer_h
 #define vtkF3DRenderer_h
 
+#include <map>
```

Comment:
std include should be put after vtk includes, with an empty lines in betweem

Reply:
Could the formatter enforce that? something like:
```yaml
IncludeBlocks: Regroup
IncludeCategories:
  - Regex:           '".*"'
    Priority:        1
  - Regex:           '^<(vtk.*)>'
    Priority:        2
    CaseSensitive:   true
  # ... anything else ?
```

Reply:
good point, wdyt @Meakk ?

Reply:
alternative is to write a guide style, but such a doc are hard to write and maintain

Reply:
First try at a coding style doc: https://github.com/mwestphal/f3d/blob/jekyll_doc/doc/dev/CODING_STYLE.md